lib/auth/style.scss

```diff
@@ -63,6 +64,22 @@
     font-weight: 400;
   }
 
+  input:focus {
+    border-color: transparent;
+    box-shadow: 0 0 0 2px $studio-simplenote-blue-50;
```

Comment:
I saw that the border is maybe expanded too much when the element is focused (see attached screenshot), does this match the designs? (I can't check them because I don't have access 😄 )
<img width="697" alt="Screenshot 2021-04-13 at 11 16 08" src="https://user-images.githubusercontent.com/14905380/114529064-e09f1600-9c49-11eb-946d-3dd0ccb1c50a.png">

I was wondering if we could set the border color with the same blue and instead of having two pixels in the box shadow, reduce it to one (see attached screenshot):
```
border-color: $studio-simplenote-blue-50;
box-shadow: 0 0 0 1px $studio-simplenote-blue-50;
```

<img width="684" alt="Screenshot 2021-04-13 at 11 19 33" src="https://user-images.githubusercontent.com/14905380/114529474-44c1da00-9c4a-11eb-97e1-171c6f89ceda.png">

<hr>

Additionally I'm not sure how the the element should look like when it's invalid and focused, now the border is set to grey but the focus color should be blue, is this right?

Here you can check how it looks like with the current changes and what I proposed:
Current changes|Proposal
--|--
<video src=https://user-images.githubusercontent.com/14905380/114528023-e516ff00-9c48-11eb-99a8-66f2fa465564.mp4>|<video src=https://user-images.githubusercontent.com/14905380/114528110-fb24bf80-9c48-11eb-85c4-f3e1e908094e.mp4>

Reply:
I like the suggestion of going with both border and box-shadow. I'm going to work on that. I have screenshots of the designs in the issue related to this #2823 under the light and dark mode toggles. 

@SylvesterWilmott how would you want the invalid state to look when focused? I had left it as it was previously, but we could update it for sure.

Reply:
>I had left it as it was previously, but we could update it for sure.

As it is currently looks good to me!